tools/reviewer/job/sync/full_github_pr.proto

```diff
@@ -68,6 +52,9 @@ message ReviewComment {
   string body = 11;
   string created_at = 12;
   string updated_at = 13;
+  Links _links = 14;
```

Comment:
**oferb's comment:**
Why does `_links` start with an `_`?
Why is `Links` plural? It has just a single link - `pull_request`.

Reply:
`_links` was started with a `_` because of the GitHub's response has the field with name `_links`. It needed for conversion from JSON to proto. It has already fixed. I just replace `_links` into `link` before parsing the response. The part of GitHub response(https://developer.github.com/v3/pulls/comments/#list-comments-in-a-repository):
```
...

 "_links": {
      "self": {
        "href": "https://api.github.com/repos/octocat/Hello-World/pulls/comments/1"
      },
      "html": {
        "href": "https://github.com/octocat/Hello-World/pull/1#discussion-diff-1"
      },
      "pull_request": {
        "href": "https://api.github.com/repos/octocat/Hello-World/pulls/1"
      }
    }
```